### PR TITLE
Ogury - 5.0.9.2

### PR DESCRIPTION
**Pull request by chauduyphanvu**

### CODE REVIEW

File: Ogury/src/main/java/com/mopub/mobileads/OguryAdapterConfiguration.java [hunk 1]
```diff
@@ -21,6 +21,8 @@
 import static com.mopub.common.logging.MoPubLog.AdapterLogEvent.CUSTOM_WITH_THROWABLE;
 
 public class OguryAdapterConfiguration extends BaseAdapterConfiguration {
+    public static final String CHOICE_MANAGER_CONSENT_ORIGIN = "MOPUB";
```
Comment (guiom-vitte) [inline Ogury/src/main/java/com/mopub/mobileads/OguryAdapterConfiguration.java:24]: Is there any way to know which CMP the consent is coming from? The consent doesn't always come from the MoPub CMP, right?

Comment (chauduyphanvu) [reply Ogury/src/main/java/com/mopub/mobileads/OguryAdapterConfiguration.java:24]: There isn't. This consent is collected by MoPub via the MoPub consent dialog or APIs. 

Publishers who integrate/mediate any other ad networks that require consent statuses to be separately forwarded to them will need to respond to those networks' consent dialogs outside of this flow. For that use case, the consent is not collected and managed by MoPub.

Comment (guiom-vitte) [reply Ogury/src/main/java/com/mopub/mobileads/OguryAdapterConfiguration.java:24]: ok so let's go for that

**Review by guiom-vitte — APPROVED** [review]

File: Ogury/src/main/java/com/mopub/mobileads/OguryBanner.java [hunk 2]
```diff
@@ -61,6 +63,11 @@ protected void load(@NonNull Context context, @NonNull AdData adData) {
         Preconditions.checkNotNull(context);
         Preconditions.checkNotNull(adData);
 
+        if (OguryAdapterConfiguration.initialized()) {
+            OguryChoiceManagerExternal.setConsent(MoPub.canCollectPersonalInformation(),
```
Comment (chauduyphanvu) [inline Ogury/src/main/java/com/mopub/mobileads/OguryBanner.java:67]: @guiom-vitte, according to the feedback on https://github.com/mopub/mopub-ios-mediation/pull/409#discussion_r673060202, should we also check for unknown consents on Android?

Comment (guiom-vitte) [reply Ogury/src/main/java/com/mopub/mobileads/OguryBanner.java:67]: Yes, please.

**Comment by chauduyphanvu** [comment]
@chiinau, could you help with the review? Thanks!

**Comment by chiinau** [comment]
@chauduyphanvu Seems good to me 👍🏻

**Pull request merged**